sda/cmd/download/swagger_v1.yml

```diff
@@ -0,0 +1,265 @@
+        - in: query
+          name: filePath
+          schema:
+            type: string
+          required: false
+          description: File path of the file to download. It is an error to supply both filePath and fileId
+        - in: query
+          name: end
+          description: 'End of requested data in the form of the 0-index of the first octet not included in the
+                                 response. That is, data delivered will have octet indices in the closed-open set
+                                 [start,end). Defaults to end of file if not set'
+          required: false
+          schema:
+            minimum: 0
+            type: integer
+        - in: query
+          name: start
+          required: false
+          schema:
+            default: 0
+            minimum: 0
+            type: integer
+      responses:
+        "200":
+          content:
+            application/octet-stream:
+              schema:
+                type: string
+                format: binary
+          description: Successful operation
+        "206":
+          content:
+            application/octet-stream:
+              schema:
+                type: string
+                format: binary
+          description: Successful partial file delivery
+        "400":
+          description: Bad request i.e start larger than end
+        "401":
+          description: Authentication failure
+        "403":
+          description: The user does not have access to the dataset
+        "416":
+          description: Unsatisfiable range i.e. end larger than file size
+        "500":
+          description: Internal application error
+      security:
+        - bearerAuth: []
+  /file/{fileId}:
+    get:
+      description: Returns the requested file to the user
+      parameters:
+        - in: header
+          description: "Public key used to re-encrypt the file header with"
+          name: public_key
+          required: true
+          schema:
+            type: string
+        - in: path
+          name: fileId
+          schema:
+            type: string
+          required: true
+          description: Stable ID of the file to download
+        - in: header
+          name: Range
+          required: false
+          schema:
+            type: string
+          description: 0-indexed byte range of the file that is to be delivered.
+          example: "Range: bytes=20-200, 300-400"
+      responses:
+        "200":
+          content:
+            application/octet-stream:
+              schema:
+                type: string
+                format: binary
+          description: Successful operation
+        "206":
+          content:
+            application/octet-stream:
+              schema:
+                type: string
+                format: binary
+          description: Successful partial file delivery
+        "400":
+          description: Bad request i.e start larger than end
```

Comment:
Is `start lager than end` the only condition for an `400` error? If not, better to use `e.g.` instead of `i.e.`.

If yes, change `i.e` -> `i.e.`